main.nf

```diff
@@ -103,6 +109,49 @@ else {
   is_staging = false
 }
 
+// Extract center list for MAF processing
+if (major_release == "TEST"){
+  all_centers = ["SAGE", "TEST", "GOLD"]
+} else {
+  all_centers = ["JHU","DFCI","GRCC","NKI","MSK","UHN","VICC","MDA","WAKE","YALE","UCSF","CRUK","CHOP","VHIO","SCI","PROV","COLU","UCHI","DUKE","UMIAMI"]
+}
+if (params.maf_centers = "ALL") {
+  maf_center_list = all_centers
+}
+
+def process_maf_helper(maf_centers, ch_project_id, maf_center_list, create_new_maf_db) {
+    /**
+  * Processes MAF files for a given center list.
+  *
+  * @param maf_centers          Parameter containing the centers to be processed, can be "ALL" or a comma-separated list
+  * @param ch_project_id        Channel with project ID
+  * @param maf_center_list          List of centers to be processed converted from params.maf_centers
+  * @param create_new_maf_db    Boolean flag to create new DB
+  * @return                     A collect output of the MAF process
+  */
+
+  // Create a channel from the list of centers
+  ch_maf_centers = Channel.fromList(maf_center_list)
+  // placeholder for previous output
+  previous = false
+  // If maf_centers is "ALL", we will process all centers in the maf_center_list
+  if (maf_centers == "ALL") {
+    // Create a channel to indicate whether it's the first center or not
+    ch_maf_centers = ch_maf_centers.branch { v ->
+        first: v == maf_center_list[0]
+        remaining: v != maf_center_list[0]
+    }
+    // Process the first center with createNewMafDb as true
+    process_maf_col1 = process_maf(previous, ch_project_id, ch_maf_centers.first, true).collect()
+    // Process the rest with createNewMafDb as false
+    return process_maf_reuse(process_maf_col1, ch_project_id, ch_maf_centers.remaining, false).collect()
```

Comment:
nit: similar, could use something like `process_maf_remaining_centers` instead of `process_maf_reuse`